accounting/filters/OsgScheddCpuRemovedFilter.py

```diff
@@ -0,0 +1,400 @@
+        if i.get("JobStatus",4) != 3:
+            return
+
+        # Filter out jobs that did not run in the OS pool        
+        if i.get("LastRemotePool", self.schedd_collector_host(schedd)) != self.collector_host:
+            return
+
+        # Get list of attrs
+        filter_attrs = DEFAULT_FILTER_ATTRS.copy()
+
+        # Count number of history ads (i.e. number of unique job ids)
+        o["_NumJobs"].append(1)
+
+        # Do filtering for scheduler and local universe jobs
+        univ = i.get("JobUniverse", 5)
+        o["_NumSchedulerUnivJobs"].append(univ == 7)
+        o["_NumLocalUnivJobs"].append(univ == 12)
+        o["_NoShadow"].append(univ in [7, 12])
+
+        # Compute badput fields
+        if (
+                univ not in [7, 12] and
+                i.get("NumJobStarts", 0) > 1 and
+                i.get("RemoteWallClockTime", 0) > 0 and
+                i.get("RemoteWallClockTime") != i.get("CommittedTime")
+            ):
+            o["_BadWallClockTime"].append(i["RemoteWallClockTime"] - i.get("CommittedTime", 0))
+            o["_NumBadJobStarts"].append(i["NumJobStarts"] - 1)
+        else:
+            o["_BadWallClockTime"].append(0)
+            o["_NumBadJobStarts"].append(0)
+
+        # Add attr values to the output dict, use None if missing
+        for attr in filter_attrs:
+            o[attr].append(i.get(attr, None))
+
+    def user_filter(self, data, doc):
+
+        # Get input dict
+        i = doc["_source"]
+
+        # Get output dict for this user
+        user = i.get("User", "UNKNOWN") or "UNKNOWN"
+        o = data["Users"][user]
+        
+        # Filter out jobs that were not removed
+        if i.get("JobStatus",4) != 3:
+            return
+
+        # Filter out jobs that did not run in the OS pool
+        schedd = i.get("ScheddName", "UNKNOWN") or "UNKNOWN"
+        if i.get("LastRemotePool", self.schedd_collector_host(schedd)) != self.collector_host:
+            return
+
+        # Add custom attrs to the list of attrs
+        filter_attrs = DEFAULT_FILTER_ATTRS.copy()
+        filter_attrs = filter_attrs + ["ScheddName", "ProjectName"]
+        
+        # Count number of history ads (i.e. number of unique job ids)
+        o["_NumJobs"].append(1)
+
+        # Do filtering for scheduler and local universe jobs
+        univ = i.get("JobUniverse", 5)
+        o["_NumSchedulerUnivJobs"].append(univ == 7)
+        o["_NumLocalUnivJobs"].append(univ == 12)
+        o["_NoShadow"].append(univ in [7, 12])
+
+        # Compute badput fields
+        if (
+                univ not in [7, 12] and
+                i.get("NumJobStarts", 0) > 1 and
+                i.get("RemoteWallClockTime", 0) > 0 and
+                i.get("RemoteWallClockTime") != i.get("CommittedTime")
+            ):
+            o["_BadWallClockTime"].append(i["RemoteWallClockTime"] - i.get("CommittedTime", 0))
+            o["_NumBadJobStarts"].append(i["NumJobStarts"] - 1)
```

Comment:
All `RemoteWallClockTime` and `NumJobStarts` are "badput" here, unless `CommittedTime > 0`.
```suggestion
        if (
                univ not in [7, 12] and
                i.get("NumJobStarts", 0) > 0 and
                i.get("RemoteWallClockTime", 0) > 0 and
                i.get("RemoteWallClockTime") != i.get("CommittedTime")
            ):
            if i.get("CommittedTime", 0) > 0:
                o["_BadWallClockTime"].append(i["RemoteWallClockTime"] - i.get("CommittedTime", 0))
                o["_NumBadJobStarts"].append(i["NumJobStarts"] - 1)
            else:
                o["_BadWallClockTime"].append(i["RemoteWallClockTime"])
                o["_NumBadJobStarts"].append(i["NumJobStarts"])
```